port/board/CMakeLists.txt

```diff
@@ -14,23 +14,14 @@ if(${BOARD} MATCHES "^(afc-bpm)$")
   if(${VERSION} MATCHES "(3.0)")
     message(FATAL_ERROR "${BoldRed}AFCv3.0 is not supported anymore. Use -DVERSION=3.1 instead.${ColourReset}")
   elseif(${VERSION} MATCHES "(3.1)")
-    add_subdirectory(afc-bpm/v3_1)
-  endif()
-
-elseif(${BOARD} MATCHES "^(afc-fofb-ctrl)$")
-
-  if(NOT VERSION)
-    message(FATAL_ERROR "${BoldRed}AFC board version was not selected! You must pass the flag -DVERSION=<major>.<minor>${ColourReset}")
-  endif()
-
-  message(STATUS "Selected Board version: ${VERSION}")
-  if(${VERSION} MATCHES "(3.1)")
-    add_subdirectory(afc-fofb-ctrl/v3_1)
+    add_subdirectory(afc-v3)
+  elseif(${VERSION} MATCHES "(4)")
```

Comment:
To keep this consistent, use AFC version '4.0' instead of '4' (also to accommodate future versions).